src/core/layout/qgslayoutitemmap.cpp

```diff
@@ -1767,6 +1767,17 @@ void QgsLayoutItemMap::mapThemeChanged( const QString &theme )
     mCachedLayerStyleOverridesPresetName.clear(); // force cache regeneration at next redraw
 }
 
+void QgsLayoutItemMap::currentMapThemeRenamed( const QString &theme, const QString &newTheme )
+{
+  if ( mFollowVisibilityPreset )
```

Comment:
Not sure that this outer condition is required -- I think we'd still want to update the theme name, even if it's been temporarily disabled for the map

Reply:
I was pretty sure (months ago) that without this condition, the followed map theme renaming was not propagated here. Tested again (with and without) and the renaming is always done so I removed.